Packs/RecordedFuture/pack_metadata.json

```
@@ -2,7 +2,7 @@
     "name": "RecordedFuture v2",
     "description": "Recorded Future App v2",
     "support": "partner",
-    "currentVersion": "1.1.3",
+    "currentVersion": "2.2.0",
```

```suggestion
    "currentVersion": "1.1.4",
```
The version jump can be confusing for customers.

---

So it should not be the same as we have in RecordedFuture.py ?
https://github.com/demisto/content/pull/14140/files#diff-70a479bb2ddd21051961ac8828ca0b2f166da1bc512bf9de49aa21a1715c8765L17-R17